test/Governance.t.sol

```diff
@@ -109,35 +82,6 @@ abstract contract GovernanceTest is Test {
         initialInitiatives.push(baseInitiative2);
         governance.registerInitialInitiatives(initialInitiatives);
 
-        governanceInternal = new GovernanceInternal(
-            address(lqty), address(lusd), address(stakingV1), address(lusd), config, initialInitiatives
-        );
-    }
-
-    // should not revert under any input
-    function test_averageAge(uint120 _currentTimestamp, uint120 _timestamp) public {
-        uint120 averageAge = governanceInternal.averageAge(_currentTimestamp, _timestamp);
-        if (_timestamp == 0 || _currentTimestamp < _timestamp) {
-            assertEq(averageAge, 0);
-        } else {
-            assertEq(averageAge, _currentTimestamp - _timestamp);
-        }
-    }
-
-    // should not revert under any input
-    function test_calculateAverageTimestamp(
```

Comment:
Removing harmful test. It _should_ very much revert when passed nonsensical arguments.

The average timestamp is a weighted average of the form:
```
t_avg = SUM_i(t_i * lqty_i) / SUM_i(lqty_i)
```

It is always legal to _add_ any number of new `(t_j, lqty_j)` pairs to an existing weighted average, in which case the new average will be:
```
t_avg' = (t_avg * SUM_i(lqty_i) + SUM(t_j * lqty_j)) / (SUM_i(lqty_i) + SUM_j(lqty_j))
       = (t_avg * SUM_i(lqty_i) + [SUM(t_j * lqty_j) / SUM_j(lqty_j)]) / (SUM_i(lqty_i) + SUM_j(lqty_j))
```

> **Note**: the parameters of the function `_calculateAverageTimestamp()` correspond to the following terms:
> - `_prevOuterAverageTimestamp` => `t_avg`
> - `_newInnerAverageTimestamp` => `SUM(t_j * lqty_j) / SUM_j(lqty_j)`
> - `_prevLQTYBalance` => `SUM_i(lqty_i)`
> - `_newLQTYBalance` => `SUM_i(lqty_i) + SUM_j(lqty_j)`

However, when _removing_ previously allocated amounts, even if we allow partial deallocation, the parameter `_newInnerAverageTimestamp` itself _must_ be a valid weighted average of the form:
```
_newInnerAverageTimestamp = SUM_i(t_i * lqtyDecrease_i) / SUM_i(lqtyDecrease_i)
```
where `forall i. lqtyDecrease_i <= lqty_i`. Failing to adhere to this constraint _can_ and _should_ result in reverts.